### ISSUE TITLE
Play Store Release is Gone?

### ISSUE THREAD

**Issue by Sanrax** [equeim/tremotesf-android#320]
Seems like the app disappeared from the Play Store and can no longer be found.

Are there any plans to reupload the app to Play Store so we can receive app updates through the Play Store again?

**Comment by equeim** [comment]
Yeah, my developer account was suspended due to issues with Google's new verification requirements. I'm trying to solve this but it's going slowly. If I make a new release before that it will be available here on GitHub Releases page and F-Droid.
